src/calculateRentalCost.js

```diff
@@ -5,6 +5,25 @@
  */
 function calculateRentalCost(days) {
   // write code here
+  let costOfrent = 40 * days;
+  const longTermThreshold = 7;
+  const midTermThreshold = 3;
+  const longTermDiscount = 50;
+  const midTermDiscount = 20;
+
+  if (days >= longTermThreshold) {
+    costOfrent = costOfrent - longTermDiscount;
+
+    return costOfrent;
+  }
+
+  if (days >= midTermThreshold) {
+    costOfrent = costOfrent - midTermDiscount;
+
+    return costOfrent;
```

Comment:
Similarly to the previous comment, you can return the discounted price directly without reassigning to `costOfrent`. This is also in line with checklist item 5, which advises against adding `else` after `if` with `return`.